includes/templates/responsive_classic/templates/tpl_modules_products_new_listing.php

```diff
@@ -18,7 +18,22 @@
 
   if ($products_new_split->number_of_rows > 0) {
     $products_new = $db->Execute($products_new_split->sql_query);
+    require DIR_WS_CLASSES . 'ProductConfigurationSwitch.php';
     while (!$products_new->EOF) {
+      // Rebuild switch object only if not set or different prod type
+      if (!isset($config_switches)) { 
+        $config_switches = new ProductConfigurationSwitch($products_new->fields['products_id']);
+      } else {
+         if ($products_new->fields['products_type'] != $config_switches->getProductsType()) {
+          $config_switches = new ProductConfigurationSwitch($products_new->fields['products_id']);
+         } 
+      }
+      $flag_show_model = $config_switches->getSwitch('model');
+      $flag_show_weight = $config_switches->getSwitch('weight');
+      $flag_show_quantity = $config_switches->getSwitch('quantity');
+      $flag_show_date_added = $config_switches->getSwitch('date_added');
+      $flag_show_manufacturer = $config_switches->getSwitch('manufacturer');
+      $flag_show_show_free_shipping_image = $config_switches->getSwitch('ALWAYS_FREE_SHIPPING_IMAGE_SWITCH');
```

Comment:
is there an extra `_show` in here?  ie, `$flag_show_free_shipping_image` or ideally as commented below, `$show_free_shipping_image`.

Reply:
There is, thank you.  Fixed.